tests/visualization_tests/matplotlib_tests/test_optimization_history.py

```diff
@@ -82,7 +88,11 @@ def objective(trial: Trial) -> float:
         study.optimize(objective, n_trials=3)
     figure = plot_optimization_history(studies)
     assert len(figure.get_lines()) == n_studies
-
+    assert list(figure.get_lines()[0].get_xdata()) == [0, 1, 2]
+    if direction == "minimize":
+        assert list(figure.get_lines()[0].get_ydata()) == [1.0, 1.0, 0.0]
+    else:
+        assert list(figure.get_lines()[0].get_ydata()) == [1.0, 2.0, 2.0]
```

Comment:
I think we should test not only the first study but also second and third studies.
```suggestion
    for i in range(3):
        assert list(figure.get_lines()[i].get_xdata()) == [0, 1, 2]
        if direction == "minimize":
            assert list(figure.get_lines()[i].get_ydata()) == [1.0, 1.0, 0.0]
        else:
            assert list(figure.get_lines()[i].get_ydata()) == [1.0, 2.0, 2.0]
```